### ISSUE TITLE
Decoding README.md fails if your locale is set to something odd.

### ISSUE THREAD

**Issue by mattgibbs** [slaclab/pydm#435]
Andrew Starrit found this one:  When he tries to install PyDM, setup.py attempts to decode README.md as ascii, and crashes.  This has something to do with his locale, and therefore his default string encoding.

You could argue that this is not really a bug in PyDM, but just a configuration problem on the user end.  Still, I think we should just limit README.md to ascii characters to avoid this.
 
Here's the stack trace:
```
(venv) [starritt@redgum melbourne2018workshop]$ 
Collecting pydm
  Using cached https://files.pythonhosted.org/packages/a3/6b/f037099b3ce9a6236b9831ec4155d8085773823f4a835ffc2d3027a21939/pydm-1.6.0.tar.gz
	Complete output from command python setup.py egg_info:
	Traceback (most recent call last):
  	File "<string>", line 1, in <module>
  	File "/tmp/pip-install-yv9m14z4/pydm/setup.py", line 14, in <module>
    	long_description = f.read()
  	File "/home/starritt/ws/melbourne2018workshop/venv/lib64/python3.6/encodings/ascii.py", line 26, in decode
    	return codecs.ascii_decode(input, self.errors)[0]
	UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 885: ordinal not in range(128)
    
	----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-yv9m14z4/pydm/
(venv) [starritt@redgum melbourne2018workshop]$
```

**Comment by hhslepicka** [comment]
Interesting... We can look into fixing that.

**Comment by mattgibbs** [comment]
Closed by #437
